src/popover/arrow.scss

```diff
@@ -71,11 +71,7 @@
   }
 
   &-inner {
-    inset-block-start: 2px;
-
-    &.refresh {
-      inset-block-start: 3px;
-    }
+    inset-block-start: calc(awsui.$border-width-field + 1px);
```

Comment:
I would not consider popover a form field component. I would treat it similar to alert, and create its own token or use a generic dropdown/popup token for dropdown and popover.